src/pybind/mgr/dashboard/frontend/angular.json

```diff
@@ -28,7 +28,9 @@
               "src/styles.scss"
             ],
             "scripts": [
-              "node_modules/chart.js/dist/Chart.bundle.js"
+              "node_modules/angular/angular.js",
+              "node_modules/chart.js/dist/Chart.bundle.js",
+              "node_modules/angular-chart.js/dist/angular-chart.js"
```

Comment:
Let's go with this one and we then may try with a more advanced [color schemer](https://www.npmjs.com/package/google-palette).